android/java/res/layout/brave_wallet_add_network.xml

```diff
@@ -217,24 +217,14 @@
 
         <androidx.appcompat.widget.AppCompatButton
             android:id="@+id/add"
+            style="@style/BraveWalletButton"
```

Comment:
`BraveWalletButton` uses the same padding attributes as before

```
        <item name="android:paddingStart">16dp</item>
        <item name="android:paddingEnd">16dp</item>
        <item name="android:paddingTop">8dp</item>
        <item name="android:paddingBottom">8dp</item>

```
so likely the padding attributes needs to be overridden here to reduce the size of the button. WDYT?